.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,6 +14,19 @@
   * Encryption
   * Audit trails
 - [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
+- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
+* Guidelines for new automation tests:
```

Comment:
Are these guidelines in a separate doc somewhere that can be linked to? I feel like this is an unprecedented amount of content for the checklist.

Reply:
👍  to Brian's comment. These should all be in a README in the automation directory since we now open up that as a distinct and separate intellij project.

Reply:
I had a similar concern. I'll move it. Thanks!